### DISCUSSION TITLE
Support rendering MP4 files

### DISCUSSION THREAD

**Discussion by mostaphaRoudsari** [GitbookIO/community#86] [Feature Requests]
### What's your feature/idea?

This might be already supported but I couldn't find a way to include MP4 animations in Gitbook. If it is already possible, then it will be nice to add it to your documentation and we can close this issue.

Otherwise, probably the easiest way to support the [HTML video tag](https://www.w3schools.com/tags/tag_video.asp). We generate the documentation in Markdown format and we don't really need this feature to be available from the UI. For now, we have to convert all the animations to the less efficient GIF format.

### What's your desired outcome?

To be able to include MP4 videos in our documentation.

### What's the impact of this for you?

- [ ] Critical
- [X] High
- [ ] Medium
- [ ] Low

**Comment by aikoii** [comment]
Hi @mostaphaRoudsari!

I am afraid we do not support videos hosted on our platform.

What we would suggest is that you upload the file to YouTube/other popular platforms or store that MP4 in a place where it can be publicly available (ex: Google Drive) and embed it in GitBook by pasting its URL and pressing Enter. The workflow is a bit better, and you won't have as many limitations on file size. 

We use Iframely for [embedded content](https://docs.gitbook.com/tour/editor/blocks/embed-a-url) on GitBook. One way you can test the link to see if it's embeddable is by using this website: https://iframely.com/embed. If it works there, it should work in GitBook.

Hope this helps! 👋

**Comment by mostaphaRoudsari** [comment]
Hi @aikoii! Thanks. That works for me. Feel free to close this feature request. Thanks!
